modules/boss/src/main/ts/ephox/boss/api/Main.ts

```diff
@@ -6,13 +9,10 @@ import { SpecialGene } from './SpecialGene';
 import { TestUniverse } from './TestUniverse';
 import { TextGene } from './TextGene';
 import { Universe } from './Universe';
-/* eslint-disable import/order */
+
 // NON API USAGE
 // used by phoenix
-import * as Logger from '../mutant/Logger';
 // used by soldier tests
-import * as Locator from '../mutant/Locator';
-/* eslint-enable import/order */
```

Comment:
I think this order is important to retain - it looks automated since the comments were left behind

- [ ] revert